packages/core/src/indexing/postgres/migrations/1-create-model-table.ts

```diff
@@ -7,6 +7,7 @@ export async function createModelTable(dataSource: Knex, tableName: string) {
 
     table.string('stream_id').primary(`idx_${indexName}_pkey`).unique(`constr_${indexName}_unique`)
     table.string('controller_did', 1024).notNullable()
+    table.jsonb('stream_content').notNullable().defaultTo('{}')
```

Comment:
why give this a default?  Don't we expect it to be always provided?

Reply:
we do currently, but wanted to standardize around the default value thats also provided here: https://github.com/ceramicnetwork/js-ceramic/blob/413b64490cfeb1a8430ecedaaeb55f106e103e2a/packages/stream-model-instance-handler/src/model-instance-document-handler.ts#L124

'{}' in the db table is defaulting to '{}::jsonb' so already an object and not a string. Fairly agnostic to keeping the default value or removing it.

Reply:
That default there is happening at a very different level.  It's saying that if the genesis **commit** doesn't have a `data` field, the resulting **stream** still comes back with a default `{}` in its content.  The stream itself should never have empty content, and I don't think it makes sense for the database layer to apply a default content separate from the stream layer.  If we ever had code that tried to add a stream to the index without providing content, that code would be a bug.  We'd want to notice that bug rather than silently ignoring it and losing the content from the database.  I recommend taking this default out.

Reply:
Ok that makes sense. I'll adjust it and drop the default then.